File: 351002/Khmel/Task340/Publisher/DTOs/Responses/CommentResponseDto.cs
    public class CommentResponseDto
    {
        public long Id { get; set; }
        public long StoryId { get; set; }
        public string Content { get; set; } = string.Empty;
        public string Country { get; set; } = string.Empty;
        public string State { get; set; } = "PENDING";
    }

File: 351002/Komar/RV4-5-6/publisher/src/main/java/by/bsuir/publisher/dto/responses/CommentResponseDto.java
package by.bsuir.publisher.dto.responses;

import by.bsuir.publisher.dto.CommentState;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@SuperBuilder
@NoArgsConstructor
@Data
@EqualsAndHashCode
public class CommentResponseDto {
    private Long id;
    private Long newsId;
    private String content;
    private CommentState state;
}
